daiquiri/handlers.py

```diff
@@ -101,7 +101,11 @@ class TTYDetectorStreamHandler(logging.StreamHandler):
 
     def format(self, record):
         if hasattr(self.stream, "isatty"):
-            record._stream_is_a_tty = self.stream.isatty()
+            try:
+                record._stream_is_a_tty = self.stream.isatty()
+            except ValueError:
+                # Stream have been closed, usually during intepretor shutdown
```

Comment:
```suggestion
                # Stream has been closed, usually during interpretor shutdown
```

Reply:
I cannot commit in your branch :(

Reply:
my fork looks bugged, I can't push on master on my own fork :roll_eyes: